schema.js

```diff
@@ -42,6 +52,12 @@ const RootQuery = new GraphQLObjectType({
                 return db('resources').select()
             }
         },
+        recovery: {
```

Comment:
I'm glad that we got this endpoint to work. Do you think that we should also set up a GraphQL endpoint for getting a single recovery group by id or name?

Reply:
Per our frontend, they will just be mapping all the points. There's nothing setup at this point where we need to retrieve just one recovery meeting. That's something we can consider for a future iteration if necessary.